zebra-script/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "zebra-script"
-version = "1.0.0-beta.20"
+version = "1.0.0-beta.21"
```

Comment:
Can we please update zcash_script before doing this release?
- #6214

We might need to un-freeze all PRs to get that PR to merge, because merge freeze cancels mergify PRs as well.

Reply:
I unfroze the repo. I'll keep track of all newly merged PRs.

Reply:
Thanks, it shouldn't take long!

Reply:
PR https://github.com/ZcashFoundation/zebra/pull/6214 has merged.